packages/manager/modules/pci/src/projects/project/storages/databases/database/general-information/general-information.routing.js

```diff
@@ -216,11 +221,14 @@ export default /* @ngInject */ ($stateProvider) => {
       stopPollingNodesStatus: /* @ngInject */ (
         DatabaseService,
         database,
-      ) => () =>
+      ) => () => {
```

Comment:
This change (adding curly braces) is actually not required

Reply:
Done here: https://github.com/ovh/manager/pull/5488/commits/45d16da89ff2c70503a9a111c3fd5996a5eecbe1